index.html

```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
  </div>
 </section>
 
+<section>  
+  <div class="content" id="cvrm">
+    <h1>Predictive Policing</h1>
+    <p>Since 2012, the CPD has used a statistical model to predict which individuals are most likely to be involved in shooting incidents. The model, developed in collaboration with the Illinois Institute of Technology, was initially called the Strategic Subject List (SSL) and was later revised and renamed the Crime and Victimization Risk Model (CVRM). This model was quietly <a id="cvrmlink" href="https://igchicago.org/2020/01/23/advisory-concerning-the-chicago-police-departments-predictive-risk-models/">decommissioned</a> by the CPD in 2019. </p>
+    <div class="row">
+    	<div class="col-md-6"></div>
+        <div class="col-md-6"><img src="img/CVRMimage.png" alt="Socmint" class="img-rounded"></div>
+    </div>
+ </div>
+</section>
+
+<section class="nonheading">  
+  <div class="content">
+  	<div class="row">
+    <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
+    <p>The CVRM was meant to play a key role in CPD’s new <a href="https://www.rand.org/pubs/research_reports/RR3242.html">Strategic Decision and Support Centers (SDSCs)</a>, real time crime centers which are now located in all but two of Chicago’s 22 districts. Officers could also view at risk individuals identified by the CVRM on a phone app.</p>
+
+    <h3> <span class="glyphicon glyphicon-piggy-bank"></span> Costs</h3> 
+    <p>CPD received $3.8 million in federal grants to develop the SSL and CVRM.</p>
+    </div>
+	<div class="row">
+		<div class="col-md-6"></div>
+		<div class="col-md-6"><img src="img/SSL-Dash.jpg" style="width:100%"><h5>A screenshot of an individual's risk classification produced by the SSL. Obtained via FOIA request by <a href="https://southsideweekly.com/predictive-policing-long-road-transparency/">South Side Weekly</a>.</div>
+	</div>
+  	<div class="row">
+    <h3> <span class="glyphicon glyphicon-eye-open"></span> Capabilities </h3>
+    <p> The CVRM outputs a risk classification for anyone arrested in Chicago over the past four years (over 300,000 people). The model’s output is based on risk factors such as a subject’s age and involvement in various types of crimes, with more recent incidents being given a higher weight than older ones. Individuals’ scores are also influenced by the scores of co-arestees. IIT provided a <a href="https://home.chicagopolice.org/wp-content/uploads/2019/01/FACT-SHEET-Crime-and-Victimization-Risk-Model-1.pdf">fact sheet</a> which describes the model in greater detail.</p>
+    <p>CVRM contributed to the <a href="https://home.chicagopolice.org/information/violence-reduction-strategy-vrs/">Custom Notification Program</a>, whereby at risk individuals and families are contacted and visited, reminded of the risk of being involved in violent crime, and connected with social services. </p>
```

Comment:
CPD and the States attorney would explicitly give people _higher_ sentences if they attempted to do a notification (source http://directives.chicagopolice.org/directives/data/a7a57bf0-1456faf9-bfa14-570a-a2deebf33c56ae59.pdf?hl=true) 

Another concern is that an attempted notifications doesn't mean a person was reached (its been told to me that sometimes they would give notification to friends, neighbors or family). This is mentioned on page 4 of the OIG report.

Reply:
Hi @ymaltsman I think we didn't address this point, that the SSL will be used for targeted enhancements.